package.json

```diff
@@ -9,21 +9,22 @@
     "node": ">=18.0.0"
   },
   "scripts": {
-    "clean": "gulp dist-clean",
-    "distclean": "gulp dist-clean && gulp build && gulp copy-production",
-    "start": "cd dist && NODE_ENV=production node index",
-    "compile": "gulp dist-clean && gulp build && gulp copy-production",
-    "dev": "gulp build && gulp copy && cd dist && node index | pino -o 'short'",
+    "clean": "rimraf dist",
+    "build": "npx tsc -b",
+    "start": "cd dist && node index.js",
+    "compile": "npx tsc -b",
     "test": "node build.js && mocha --exit"
   },
   "dependencies": {
+    "@scicatproject/scicat-sdk-ts-fetch": "^4.14.0",
     "@types/node": "^20.14.2",
     "bluebird": "^3.7.2",
     "body-parser": "^1.20.2",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
     "express": "^4.19.2",
+    "fetch": "^1.1.0",
```

Comment:
the fetch package should not be needed, as node 18+ has its own built-in fetch

Reply:
Done and tested